### PR TITLE
feat: ja translate introduction/what-is-serverless-caching

### LINKED ISSUE #203: [JP Translation] What is Serverless Caching
https://github.com/momentohq/public-dev-docs/blob/main/i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md

### PR DESCRIPTION
**Pull request by riywo**
Close #203

### CODE REVIEW

**Comment by riywo** [comment]
@poppoerika Check this out!

File: i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md [hunk 1]
```diff
@@ -1,106 +1,107 @@
...
+第一に、Momento Cache はマネージドサービスです。ソフトウェアをインストールしたり、フェイルオーバーを管理したり、バージョンを更新したりする必要はありません。これらは裏で適切に処理されているので、皆さんはご自身のアプリケーションの中心となる場所の開発と運用に専念することができます。
 
-Finally, Momento Serverless Cache has a pay-for-value pricing model. As mentioned above, you aren't selecting a specific cache and cluster size in advance, regardless of whether you use it. Momento charges based on the amount of data stored and the requests made against your cache. With this, you are in control of your pricing, and changes you make to your application have a direct impact on your bill.
+第二に、Momento Cache はキャッシュ管理に関する設定を抽象化しています。キャッシュに必要なインスタンスの数を指定することも、最大メモリサイズを考えることも必要ありません。Momento はクラウドに特化して作られていて、現代的なクラウド基盤の弾力性やスケーラビリティを最大活用しています。皆さんはキャッシュに必要なデータをどれだけでも保存することができて、Momento がそれをシームレスに取り扱います。
 
-Under this first definition of serverless, Memento Serverless Cache is the cache that fits best in the serverless ecosystem.
+最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento はキャッシュに保存されたデータ量と行われたリクエストに対して課金をします。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
```
Comment (riywo) [inline i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: nit: "Momento charges based on the amount of data stored" could miss-lead as its cost dimension is transferred data volume.

File: docs/introduction/what-is-serverless-caching.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Second, Momento Cache abstracts the decisions around cache management from you.
 
 Finally, Momento Cache has a pay-for-value pricing model. As mentioned above, you aren't selecting a specific cache and cluster size in advance, regardless of whether you use it. Momento charges based on the amount of data stored and the requests made against your cache. With this, you are in control of your pricing, and changes you make to your application have a direct impact on your bill.
 
-Under this first definition of serverless, Memento Serverless Cache is the cache that fits best in the serverless ecosystem.
+Under this first definition of serverless, Memento Cache is the cache that fits best in the serverless ecosystem.
```
Comment (poppoerika) [inline docs/introduction/what-is-serverless-caching.md:49]: thank you!

File: i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md (hunk 1 above)
Comment (NoSQLKnowHow) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: Yeah, this is old wording. Let me look at the page this is based on and we can update it.

Comment (NoSQLKnowHow) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: This should read "Momento charges for data transferred in/out of Momento Cache and Momento Topics services. Everything else is included."

Comment (poppoerika) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: @riywo thank you for your feedback!
Could you possibly update the original doc with what Kirk mentioned above?

Comment (NoSQLKnowHow) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: If it is possible, please add a link to the pricing page for more information. https://docs.momentohq.com/manage/pricing

File: i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md [hunk 3]
```diff
@@ -1,106 +1,107 @@
...
+第二に、Momento Cache は十分な量の無料枠を持っています。毎月 50 GB が無料です(詳細は[料金](./../manage//pricing)をご覧下さい)。私達のゴールは、多種多様なアプリケーションが1円も払うことなく Momento 上で実行されていることです。私たちは最高レベルに頑丈なサービスを求めるアプリケーション向けに提供する一方で、Momento を使って成長していくアプリケーションの幅広いコミュニティもサポートしていきます。
 
-## Conclusion
+## まとめ
 
-In this page, you learned how Momento Serverless Cache fits with every conception of serverless. Momento has been designed specifically for modern architectures, with a serverless operational model, an integration model that works well with Lambda-powered applications, and a sign up and billing model that works for all types of developers and teams.
+このページでは、あらゆるサーバーレスの概念においても Momento Cache が適合していることを解説しました。Momento Cache は現代的なアーキテクチャに特化して設計されており、サーバーレスな運用モデル、Lambda で作られたアプリケーションと協調できる連携モデル、そしてあらゆるタイプの開発者やチームと親和性のあるサインアップと課金モデルを持っています。
 
-If you're ready to get started with Momento Serverless Cache, be sure to check out the following materials:
+If you're ready to get started with Momento Cache, be sure to check out the following materials:
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: Forgot to remove?

File: i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md (hunk 1 above)
Comment (riywo) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: Looks good. Will update both EN and JA.

Comment (riywo) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: Updated.

File: i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md (hunk 3 above)
Comment (riywo) [reply i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md]: Updated.

**Review by poppoerika — APPROVED** [review]

**Review by poppoerika — APPROVED** [review]

**Pull request merged**